src/OpenTelemetry.Instrumentation.Http/MeterProviderBuilderExtensions.cs

```diff
@@ -37,6 +40,11 @@ public static class MeterProviderBuilderExtensions
     {
         Guard.ThrowIfNull(builder);
 
+#if NET8_0_OR_GREATER
+        return builder
+        .AddMeter("System.Net.Http")
+        .AddMeter("System.Net.NameResolution");
```

Comment:
nit: Indention looks a bit odd.

```suggestion
        return builder
            .AddMeter("System.Net.Http")
            .AddMeter("System.Net.NameResolution");
```